pkg/tbtc/moved_funds_sweep.go

```diff
@@ -82,3 +288,58 @@ func ValidateMovedFundsSweepProposal(
 
 	return nil
 }
+
+func assembleMovedFundsSweepTransaction(
+	bitcoinChain bitcoin.Chain,
+	walletPublicKey *ecdsa.PublicKey,
+	movedFundsUtxo *bitcoin.UnspentTransactionOutput,
+	walletMainUtxo *bitcoin.UnspentTransactionOutput,
+	fee int64,
+) (*bitcoin.TransactionBuilder, error) {
+	if movedFundsUtxo == nil {
+		return nil, fmt.Errorf("moved funds UTXO is required")
+	}
+
+	builder := bitcoin.NewTransactionBuilder(bitcoinChain)
+
+	totalInputValue := int64(0)
+
+	// The moved funds UTXO is always the first input.
+	err := builder.AddPublicKeyHashInput(movedFundsUtxo)
+	if err != nil {
+		return nil, fmt.Errorf(
+			"cannot add input pointing to moved funds UTXO: [%v]",
+			err,
+		)
+	}
+	totalInputValue += movedFundsUtxo.Value
+
+	// If the wallet has the main UTXO it becomes the second input.
+	if walletMainUtxo != nil {
+		err := builder.AddPublicKeyHashInput(walletMainUtxo)
+		if err != nil {
+			return nil, fmt.Errorf(
+				"cannot add input pointing to main wallet UTXO: [%v]",
+				err,
+			)
+		}
+		totalInputValue += walletMainUtxo.Value
+	}
+
+	// Add a single output transferring funds to the wallet itself.
+	outputValue := totalInputValue - fee
```

Comment:
We can use `builder.TotalInputsValue()` instead of computing `totalInputValue` manually.

Reply:
Done in 664ace242d0477c43569889b6a382c7024cd145b.